response_operations_ui/views/messages.py

```diff
@@ -231,6 +231,7 @@ def view_selected_survey(selected_survey):
     displayed_short_name = format_short_name(selected_survey)
     session['messages_survey_selection'] = selected_survey
     breadcrumbs = [{"text": displayed_short_name + " Messages"}]
+    session['totals'] = {}
```

Comment:
Why's this here?  Not sure where it's used

Reply:
Good catch , I was passing the counts via session and this was initialising the dict. But I moved it off session and missed this . Thanks